### ISSUE TITLE
Helm template reportportal.fullname refers to incorrect value

### ISSUE THREAD

**Issue by psalacha** [reportportal/kubernetes#425]
In Helm chart v. 24.1.3 you've changed some values of the chart and put them into "global" section, i.e.:
```
global:
  imageRegistry: ""
  imagePullSecrets: []
  nameOverride: ""
  fullnameOverride: ""
```

But you did not modify the "reportportal.fullname" template entirely. There is a fragment now:
```
{{- define "reportportal.fullname" -}}
{{- if .Values.fullnameOverride -}}
{{- .Values.global.fullnameOverride | trunc 63 | trimSuffix "-" -}}
{{- else -}}
...
```
which should look like this:
```
{{- define "reportportal.fullname" -}}
{{- if .Values.global.fullnameOverride -}}
{{- .Values.global.fullnameOverride | trunc 63 | trimSuffix "-" -}}
{{- else -}}
...
```

**Comment by Pink-Bumblebee** [comment]
@psalacha , thank you for highlighting. We will check.

**Comment by hlebkanonik** [comment]
Released in 24.2.0
